server/segment_rpc.go

```diff
@@ -549,3 +555,11 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 
 	return base64.StdEncoding.EncodeToString(data), nil
 }
+
+func validatePrice(sess *BroadcastSession) bool {
+	oPrice := big.NewRat(sess.OrchestratorInfo.PriceInfo.GetPricePerUnit(), sess.OrchestratorInfo.PriceInfo.GetPixelsPerUnit())
+	if BroadcastCfg.MaxPrice() != nil && oPrice.Cmp(BroadcastCfg.MaxPrice()) == 1 {
```

Comment:
We can avoid double locking by storing the result of `BroadcastCfg.MaxPrice()` in a variable instead of making 2 calls

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/992/commits/18648d78c0d532e7e1cc26d8bc1e374a560cee34